### PR TITLE
Data model implementation

### PR DESCRIPTION
**Pull request by wyusuf068**
- Implementing Ottawa's wastewater data with the Ottawa Data Model saved in `wwMeasure.csv`.

### CODE REVIEW

File: data/wastewater_virus.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-sampleDate,siteID,siteName,reportDate,covidN1_PMMV_meanNormal,covidN1_PMMV_SDNormal,covidN2_PMMV_meanNormal,covidN2_PMMV_SDNormal,Ct_mean,dataConcern
+sampleDate,siteID,siteName,reportDate,covN1_nPMMoV_meanMnNr,covN1_nPMMoV_sdNr,covN2_nPMMoV_meanMnNr,covN2_nPMMoV_sdNr,Ct_mean,dataConcern
```
Comment (DougManuel) [inline data/wastewater_virus.csv]: I updated these headers to conform with V1.1.0. 

What does Ct_mean correspond to? covN1, covN2, PPMoV?

Comment (wyusuf068) [reply data/wastewater_virus.csv]: PMMoV cycle threshold. This is the definition I found from Public Health Ontario:

> The Ct value is defined as the number of cycles of amplification (using rRT-PCR) required for the
fluorescence of a PCR product (i.e. the target/amplicon) to be detected crossing a threshold, which is
above the background signal (a low level signal that is present in the assay regardless of whether target
is present).

**Review by DougManuel — COMMENTED** [review]
1) Can you add the metrics that are used to plot the figures? 

An example is: 
`covN1covN2_nPPMoV_meanMnNr`

I can tidy the names if you are sure. This would mean writing back to them from your R code to the GH CSV.

2) Can you add the VOC data to wastewater_virus.csv

File: data/wastewater_virus.csv (hunk 1 above)
Comment (DougManuel) [reply data/wastewater_virus.csv]: Yes. but that corresponds to one of the metrics, right? do you know which metric? covN1, covN2, nPPMoV?

**Pull request merged**